### PR TITLE
Move TE strings out

### PR DESCRIPTION
**Pull request by Adaptivity**
They handled by TE now. Can be merged when https://github.com/SlimeKnights/IguanaTweaksTConstruct/issues/115 gets fixed.

### CODE REVIEW

**Comment by bonii-xx** [comment]
Has TF added them to their stuff yet?

**Comment by Adaptivity** [comment]
https://github.com/CoFH/ThermalFoundation/blob/master/src/main/resources/assets/thermalfoundation/lang/en_US.lang#L294 but ITT bug still valid.

File: resources/assets/tinker/lang/ru_RU.lang [hunk 1]
```diff
@@ -1812,13 +1703,7 @@ toolpart.LargePlate.copper=Медная большая пластина
...
 toolpart.LargePlate.alumite=Алюмитовая большая пластина
 toolpart.LargePlate.steel=Стальная большая пластина
-toolpart.LargePlate.thaumium=Таумическая большая пластина
-toolpart.LargePlate.electrum=Электрумовая большая пластина
-toolpart.LargePlate.invar=Инваровая большая пластина
-toolpart.LargePlate.lead=Свинцовая большая пластина
-toolpart.LargePlate.nickel=Никелевая большая пластина
-toolpart.LargePlate.platinum=Платиновая большая пластина
-toolpart.LargePlate.silver=Серебряная большая пластина
+toolpart.LargePlate.thaumium=Таумическая большая пластин
```
Comment (fewizz) [inline resources/assets/tinker/lang/ru_RU.lang]: пластин?

**Comment by Adaptivity** [comment]
@bonii-xx can be merged now.

**Comment by Adaptivity** [comment]
Will resubmit later.

**Pull request closed without merging**